src/utils/substrate-rpc.ts

```diff
@@ -0,0 +1,31 @@
+import { Keyring } from '@polkadot/api'
+import { cryptoWaitReady } from '@polkadot/util-crypto'
+import { type SafeEventEmitterProvider } from '@web3auth/base'
+import { encodeSecretKey } from './account'
+
+export class SubstrateRPC {
+  private provider: SafeEventEmitterProvider
+
+  constructor(provider: SafeEventEmitterProvider) {
+    this.provider = provider
+    cryptoWaitReady()
```

Comment:
use dynamic import here, so that the first load js won't include the polkadot util, because it will increase 100-300kb.